content/en/blog/wireshark.md

```diff
@@ -119,52 +118,108 @@ service PersonSearchService {
+-->
 ![wireshark_decode_as_dialog](/img/wireshark_decode_as_dialog.png)
 
-This is a screenshot of the gRPC service request message of
-[grpc_person_search_protobuf_with_image.pcapng](https://gitlab.com/wireshark/wireshark/-/wikis/uploads/f6fcdceb0248669c0b057bd15d45ab6f/grpc_person_search_protobuf_with_image.pcapng):
+### Decoding the search request message
+
+<!--
+TODO: What are the steps that the user needs to follow before the user can see
+      the decoded message as shown in the screenshot?
+-->
+Here's a screenshot of the sample's service request message:
+
+<!--
+TODO: Can you remove the commentary text (in blue) from the image. Such text is
+      best placed in the body of the post (it makes it easier to edit, and
+      eventually possibly translate into other languages :))
+
+TODO: If a users opens up the "GRPC_Message" section in that display, what will
+      be shown?
+-->
 ![wireshark_grpc_protobuf_search_request](/img/wireshark_grpc_protobuf_search_request.png)
-That shows searching for the Person objects of Jason and Lily based on their
-names.
+
+By examining the HTTP2 message header `path` field, you'll notice the URL to the
+app's service (`/tutorial.PersonSearchService`), followed by the name of the
+invoked RPC (`Search`).
+
+The `content-type`, which is set by the gRPC library, informs Wireshark that the
+HTTP2 message content is a gRPC message. By examining the decoded Protocol
+Buffers message of the sample gRPC request, you can see that the search request
+is for the names "Jason" and "Lily".
+
+### Decoding the server-streamed response
 
 Since the `Search` operation is defined as the server streaming RPC mode, the
 Person objects can be return back to client one after another:
+
+<!--
+
+TODO: The response is sent over port 51035. Does a user need to configure the
+      "decoding as" HTTP2 for this port?
+
+      It might also be worth mentioning to the reader how we know that 51035 is the response port.
+-->
 ![wireshark_grpc_protobuf_search_response](/img/wireshark_grpc_protobuf_search_response.png)
 
 You may have noticed that the `portrait_image` of bytes type is parsed as a
-PNG data. It's because we register the PNG dissector in the `"protobuf_field"`
+PNG data.
+<!--
+TODO: Question: does the user actually see the image? If so, that isn't obvious from the screenshot.
+
+FIXME: Regarding the start of the sentence above: "You may have noticed that
+       ..." The user can't notice the decoding of the image because we haven't
+       guided them in setting up the PNG dissector prior to this point.
+
+Suggestion: Actually, I suggest that we drop the details concerning the
+       setup of the image dissector, and simply refer any interested reader to
+       https://gitlab.com/wireshark/wireshark/-/wikis/Protobuf#protobuf-field-subdissectors.
+       We can simply mention that subdissectors, like the PNG dissector can be
+       registered. Let me know what you think.
```

Comment:
I agree with you. Maybe make this blog simpler, the better.

Reply:
Done